Rocket.Chat/Views/Cells/Chat/ChatMessageImageView.swift

```diff
@@ -60,5 +61,7 @@ final class ChatMessageImageView: UIView {
 
     @objc func didTapView() {
         delegate?.openImageFromCell(attachment: attachment, thumbnail: imageView)
+        labelTitle.isHidden = true
+        detail.isHidden = true
```

Comment:
Why are you hiding it on tap? When do you show it again?

Reply:
@rafaelks My suggestion was to *toggle* hide on tap on fullscreen as title and description [if displayed at all] would overlay the image. So maybe a misunderstanding :)

Reply:
@soundstorm Gotcha, but only when in fullscreen, right? In this case, the labels are on the cell.

Reply:
Yep, so a bit mixed up here :)